client/reader/start/post-preview.jsx

```diff
@@ -56,7 +56,7 @@ const StartPostPreview = React.createClass( {
 		return (
 			<article className={ labelClasses }>
 				<div className="reader-start-post-preview__featured-label">Featured Post</div>
-				<div className="reader-start-post-preview__featured-image" style={ heroStyle }></div>
+				<div className="reader-start-post-preview__featured-image is-dark" style={ heroStyle }></div>
```

Comment:
@bluefuton @TooTallNate In the mean time that we're trying to figure out luminosity detection for featured images, I've temporarily added the `is-dark` class to them.